apps/community/src/components/about/club/club-card.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import * as style from './club-card.css';
+
+interface Props {
+  name: string;
+  description: string;
+  link?: string;
+  image?: string;
+}
+
+function ClubCard({ image, name, description, link }: Props) {
+  return (
+    <div className={style.cardWrapper}>
+      <Image
+        className={style.cardImage}
+        src={image ?? 'https://placehold.co/510x255.png'}
+        width={510}
+        height={255}
+        alt={`${name} 동아리 사진`}
+      />
+      <div className={style.cardBodyWrapper}>
+        <h3 className={style.cardTitle}>{name}</h3>
```

Comment:
```suggestion
        <h2 className={style.cardTitle}>{name}</h2>
```
page-header의 title(h1) 이후 h2 헤딩태그 없이 h3가 나오고있는 것으로 보여요